src/backend/pom.xml

```diff
@@ -29,6 +29,8 @@
 				<module>libs/efiling-cso-starter</module>
 				<module>libs/efiling-bom</module>
 				<module>libs/efiling-demo-starter</module>
+				<module>libs/efiling-diligen-client</module>
+				<module>libs/efiling-diligen-client-starter</module>
```

Comment:
Might want to add these to the efiling-reviewer profile. It won't let me comment there to suggest code....